tools/ignore_untagged_notes.sh

```diff
@@ -2,7 +2,7 @@
 
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2022.
+# (C) Copyright IBM 2021, 2022.
```

Comment:
I don't think this copyright change was intended, was it? :thinking:

Reply:
I copied the whole file from Nature :-) I will update it